### PR TITLE
Moped Tables | Refine GridTable layout/design on Projects page #5028

### PR DESCRIPTION
**Pull request by sergiogcx**
Closes https://github.com/cityofaustin/atd-data-tech/issues/5028

Preview:
https://deploy-preview-194--atd-moped-main.netlify.app/

- [x] Change page title from "Projects List" to "Projects"
- [x] Make [fixed tabs](https://material-ui.com/components/tabs/#fixed-tabs) for "Search" (currently "General Search") and "Advanced Search" (currently "Filter Search") inside the search box.
- [x] Move "Download" button (see #4785) into search box.

### CODE REVIEW

**Comment by amenity** [comment]
@sergiogcx - Hoping [these tweaks](https://github.com/cityofaustin/atd-data-tech/issues/5028#issuecomment-769666674) are reasonable for this PR, but just LMK if you'd prefer them in a separate issue.

**Comment by sergiogcx** [comment]
@amenity Ready for your review again! :) it includes those tweaks in that issue. https://github.com/cityofaustin/atd-data-tech/issues/5028

**Comment by amenity** [comment]
It's sooo beautiful, @sergiogcx! Thanks for the extra refinements.

**Review by amenity — APPROVED** [review]

File: moped-editor/src/components/GridTable/GridTableSearch.js [hunk 1]
```diff
@@ -16,65 +53,311 @@ import GridTableExport from "./GridTableExport";
...
+            <Grid
+              item
+              xs={12}
+              sm={6}
+              md={4}
+              lg={2}
+              xl={2}
+              className={classes.downloadButtonGrid}
+            >
+              {query.config.showExport && (
```
Comment (mddilley) [inline moped-editor/src/components/GridTable/GridTableSearch.js:308]: Nice to see grid in action to keep the download button in place. I've been curious about using MUI for UI details like this, and it looks real nice along with those search tabs.

**Review by mddilley — APPROVED** [review]
Very cool to see this continue to evolve, and the solution for the multi-field search is super clever! 🔍  🙌

**Comment by mateoclarke** [comment]
👍

**Pull request merged**